pkg/http/server.go

```diff
@@ -130,11 +131,24 @@ func (s *server) getRouter(endpoints []Endpoint) *routing.Router {
 	router := routing.New()
 
 	for _, e := range endpoints {
-		methods := strings.Join(e.Methods, ",")
 		path := fmt.Sprintf("/%s/%s", e.Version, e.Route)
-
-		router.To(methods, path, e.Handler)
+		for _, m := range e.Methods {
+			if m == http.MethodGet {
```

Comment:
I would use [router.Handle](https://github.com/fasthttp/router/blob/7ef9a432b73c86c9d15eab41d52de2eaf74f7da7/router.go#L240). then we can remove long if-statements.

```
router.Handle(m, path,  e.Handler)
```

Reply:
Nice, I missed that method. done.